config/config.py.sandbox

```diff
@@ -4,63 +4,65 @@ use_gcp_buckets = False
 red_bucket = ''
 green_bucket = ''
 
-sqlite_db = '/finngen/library-green/finngen_R9/finngen_R9_analysis_data/annotations/fgq.r9.1.db'
-basic_info_file = '/finngen/library-red/finngen_R9/phenotype_1.0/data/finngen_R9_gt_samples_info.txt.gz'
-intensity_files_location = '/finngen/library-red/finngen_R9/cluster_plot_1.0/data/'
+sqlite_db = '/finngen/library-green/finngen_R11/finngen_R11_analysis_data/annotations/fgq.r11.1.db'
+basic_info_file = '/finngen/library-red/finngen_R11/phenotype_1.0/data/finngen_R11_gt_samples_info.txt.gz'
+intensity_files_location = '/finngen/library-red/finngen_R11/cluster_plot_1.0/data/'
 
 geojson = '/finngen/library-green/map-coordinates/finmap.geojson'
-release_version = 'DF9'
+release_version = 'DF11'
 
 vcf_files = {
     "imputed_data": [
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr1.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr2.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr3.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr4.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr5.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr6.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr7.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr8.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr9.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr10.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr11.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr12.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr13.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr14.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr15.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr16.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr17.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr18.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr19.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr20.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr21.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr22.vcf.gz',
-        '/finngen/library-red/finngen_R9/genotype_1.0/data/finngen_R9_chr23.vcf.gz'
+'gs://fgfactory-staging-input-data/genotype_browser/finngen_R11/vcfs/finngen_R11_chr1.vcf.gz',
```

Comment:
Can we use red library for the data files? Should avoid having large genotype files in many places, also less potential for syncing errors if the data is in one place (red library)

Reply:
Sorry, my bad, I uploaded a wrong config file! Sure, I'll update that to point the correct files in Sandbox!

Reply:
Updated

Reply:
Cool - I also think the files should not be in gs://fgfactory-staging-input-data/genotype_browser in any case - we are paying for duplicated data